scripts.js

```diff
@@ -0,0 +1,123 @@
+let coursesData;
+
+function fetchCourses() {
+  let parent = document.querySelector(".swiper-wrapper");
+  fetch("http://localhost:3000/courses")
```

Comment:
for writing url hardcoded  inside fetch function is not the best practice  but you can define a constant in the top of page like that " const URL = ""http://localhost:3000/courses"  "